client/src/pages/reports/Minimal.js

```diff
@@ -262,7 +263,8 @@ class ReportMinimal extends Page {
 				<img src={person.iconUrl()} alt={person.role} height={20} width={20} className="person-icon" />
 				<LinkTo person={person} isLink={false}/>
 			</td>
-				<td>{person.position && person.position.name}</td>
+				<td><LinkTo isLink={false} position={person.position} /></td>
```

Comment:
When a person has no position it will show unspecified for position, maybe it's better to add whenUnspecified="" ?

Reply:
I left it intentionally in there as we want to make people not add attendees w/o a position. I added it to an org, as it was getting a bit redundant. If anything, I would prefer to take `whenUnspecified=""` out of org links